sdk/src/wallet/account/operations/transaction/high_level/minting/mint_nfts.rs

```diff
@@ -53,40 +53,40 @@ impl MintNftParams {
 
     pub fn with_address(
         mut self,
-        address: impl TryInto<Bech32Address, Error = impl Into<crate::wallet::Error>>,
+        address: Option<impl TryInto<Bech32Address, Error = impl Into<crate::wallet::Error>>>,
     ) -> Result<Self, crate::wallet::error::Error> {
-        self.address = Some(address.try_into().map_err(Into::into)?);
+        self.address = address.map(|v| v.try_into().map_err(Into::into)).transpose()?;
         Ok(self)
     }
 
     pub fn with_sender(
         mut self,
-        sender: impl TryInto<Bech32Address, Error = impl Into<crate::wallet::Error>>,
+        sender: Option<impl TryInto<Bech32Address, Error = impl Into<crate::wallet::Error>>>,
```

Comment:
These should not accept options imo

Reply:
Why? Not even `impl Into<Option<_>>` ?

Reply:
It's just not ergonomic and you can't mix `impl Option` and `impl Bech32Like`.

Reply:
I thought you were all in for `impl Into<Option<_>>`

Reply:
But yeah, `impl Bech32Like` is definitely more important here

Reply:
so we stay with `Option<impl Bech32Like>` or `impl Bech32Like` ?

Reply:
Trust me, if I could have both I would

Reply:
ok, as has been agreed, I've removed the `Option` from the `with_*` methods